docusaurus/src/components/Chip.module.css

```diff
@@ -0,0 +1,13 @@
+.chip {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0.125rem 0.375rem;
+  background-color: var(--color-grey-100);
+  border-radius: 0.625rem;
+  font-size: 0.625rem;
+  line-height: 0.8rem;
+  text-transform: uppercase;
+  font-weight: var(--ifm-font-weight-semibold);
+  color: #252537
```

Comment:
Did you explicitly make this not point to a color variable because you wanted it to be the same across light and dark mode?

If so, I'd recommend still setting this to a new color variable name and making it the same value across both, just to keep the consistency of having all of our color values in one file and in the same format